docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -1870,6 +1870,20 @@ In order to be able to debug the Identity Networking Policy you can launch the f
  term mon
  debug pre all
 
+DHCP Option 82
+++++++++++++++
+
+In order to enable the dhcp option 82 you need to configure add configuration parameters in the Cisco switch.
```

Comment:
do we actually need to specify "the Cisco switch" since we are under the Cisco section ?
It should be:
In order to enable the DHCP option 82, you need to add the followings parameters.